### DISCUSSION TITLE
Support for CITEXT

### DISCUSSION THREAD

**Discussion by justindotpub** [electric-sql/electric#964] [Feature requests]
Would it be possible to support [CITEXT](https://www.postgresql.org/docs/current/citext.html), perhaps using SQLite's `COLLATE NOCASE`?

**Comment by linear** [comment]
<p><a href="https://linear.app/electric-sql/issue/VAX-1678/support-for-citext">VAX-1678 Support for CITEXT</a></p>

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
